_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,603 @@
+
+  The segwit soft fork was released with [BIP9][] activation
+  parameters.  A few miners quickly began signaling readiness but support
+  peaked far below the 95% threshold.  Some Bitcoin users felt
+  miners were illegitimately delaying a useful new feature and went on
+  to develop the mandatory activation that became [BIP148][].  The final
+  form of BIP148 specified rejecting any
+  blocks that didn't signal support for segwit starting on a
+  certain date.
+
+  Software implementing BIP148 created the risk of a consensus failure.  If miners failed to
+  signal support for segwit, and most users continued to consider those
+  miners' blocks as valid, users of BIP148 might accept bitcoins that
+  would be invalid from the perspective of most other Bitcoin users.  Alternatively, if
+  most users of Bitcoin followed the BIP148 rules---but miners still
+  produced many BIP148-invalid blocks---those users who didn't enforce
+  BIP148 could accept bitcoins that the BIP148 users would consider
+  invalid.  Only if all users followed the same rules, or most mining
+  hashrate followed the BIP148 rules, would safety be assured.
+
+  One way to reduce this risk was to give more users time to upgrade to
+  a node that would mandate activation of segwit.  BIP148 couldn't do
+  this since its goal of triggering the existing BIP9 segwit deployment
+  meant it had to force miner signaling to start long before segwit's
+  BIP9 timeout date.  As an alternative in case BIP148 failed to gain
+  sufficient support, [BIP149][] was proposed to give users another year
+  to upgrade.  BIP149 never gained much public support, but it did
+  result in the creation of [BIP8][], which would see additional
+  discussion in subsequent years.
+
+  Around the time BIP148 began receiving significant public support,
+  several miners, exchanges, and other businesses signed their support
+  for a two-step proposal that started with activation of segwit in a
+  way that would remain in consensus with nodes supporting BIP148.  This first stage was proposed
+  in [BIP91][], which was a modification of the BIP9 rules.  Miners used
+  a BIP9 versionbit to signal whether they would enforce a temporary
+  rule to reject any blocks not signaling the BIP141/143 segwit
+  versionbit.  Compared to BIP9, the BIP91 threshold was reduced from
+  95% to 80% and its monitoring and lockin period lengths were reduced
+  from 2,016 blocks to 336 blocks.
+
+  BIP91 locked in and activated almost as soon as was possible.
+  Subsequently, BIP141/143 locked in and activated.  BIP148 mandatory
+  signaling expired when BIP141/143 locked in.  The second stage of
+  the proposal from miners, exchanges, and other businesses required a
+  hard fork; spokespersons for the signers eventually withdrew their
+  proposal after vocal opposition from a large number of individual
+  users and several businesses.
+
+  It remains a subject of debate exactly how much each of the above
```

Comment:
Seems like the only "debate" against BIP148's success is ego-centric refusal of some opponents to admit they were wrong. Late 2017, it was widely acknowledged, even by people who didn't believe in BIP148, that it had succeeded.

Reply:
I strongly agree that some people really do seem to be egocentrically invested in pushing a particular narrative.